<ui version="4.0" >
 <class>AdapterWidget</class>
 <widget class="QWidget" name="AdapterWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>251</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="infoGroupBox" >
     <property name="font" >
      <font>
       <underline>true</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="title" >
      <string>Information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="font" >
           <font>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="text" >
           <string>MAC Address: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="macLabel" >
          <property name="font" >
           <font>
            <underline>false</underline>
           </font>
          </property>
          <property name="text" >
           <string comment="KDE::DoNotExtract">TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="font" >
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="title" >
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="font" >
             <font>
              <underline>false</underline>
             </font>
            </property>
            <property name="text" >
             <string>Adapter Name: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nameLineEdit" >
            <property name="font" >
             <font>
              <underline>false</underline>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="font" >
             <font>
              <underline>false</underline>
             </font>
            </property>
            <property name="text" >
             <string>Mode: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="modeComboBox" >
            <property name="font" >
             <font>
              <underline>false</underline>
             </font>
            </property>
            <item>
             <property name="text" >
              <string>hidden</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>discoverable</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label_5" >
            <property name="font" >
             <font>
              <underline>false</underline>
             </font>
            </property>
            <property name="text" >
             <string>Discoverable Timeout:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="timeoutLabel" >
            <property name="font" >
             <font>
              <underline>false</underline>
             </font>
            </property>
            <property name="text" >
             <string>none</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="timeoutHorizontalSlider" >
          <property name="minimum" >
           <number>60</number>
          </property>
          <property name="maximum" >
           <number>1860</number>
          </property>
          <property name="singleStep" >
           <number>60</number>
          </property>
          <property name="pageStep" >
           <number>120</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
